css/style.css

```
@@ -131,7 +135,7 @@ img {
 .header-logo {
   position: absolute;
   width: 138px;
-  justify-self: center;
+  margin-left: 571px;
```

При позиционировании элементов абсолютом используются правила **top/right/bottom/left**
```suggestion
  left: 571px;
```